jobbergate-docs/src/developer_guide/ci.rst

```diff
@@ -2,4 +2,76 @@
  Continuous Integration
 ========================
 
-TODO: Write this in concert with Felipe
+Jobbergate uses `GitHub actions <https://github.com/omnivector-solutions/jobbergate/actions>`_ as continuous integration tools. They are described in details in this page.
+
+-------------------
+ Quality Assurance
+-------------------
+
+A dedicated GitHub action is included at Jobbergate's repository 
+(`test_on_push.yaml <https://github.com/omnivector-solutions/jobbergate/blob/main/.github/workflows/test_on_push.yaml>`_)
+to run the quality assurance tools for all sub-projects at once.
+It includes unit tests, code coverage, linters, code formatters, and static code checkers,
+that are detailed in the section :any:`Quality Assurance Tools`.
```

Comment:
```suggestion
They include unit tests, code coverage, linters, code formatters, and static type checkers.
Each is detailed in the section :any:`Quality Assurance Tools`.
```

Reply:
solved in [58f2f91](https://github.com/omnivector-solutions/jobbergate/pull/144/commits/58f2f918da90e960892e55f68c4e15734043d298).